WRLGS-11 Utils to timestamp stderr logs

BourgoisMickael opened the pull request:
Cherry-picked #167 

> [!CAUTION]
> No CI running tests on this hotfix PR :thinking:

BourgoisMickael committed:
WRLGS-11: Generic function for timestamp in stderr

(cherry picked from commit dd2d71175dae11ece235d17b2301b7d4e537271a)

BourgoisMickael committed:
WRLGS-11: Test stderrUtils

(cherry picked from commit da2224fbf7766f38812d980bc8183f8fd7bbc8ad)

BourgoisMickael committed:
WRLGS-11: Bump package version hotfix 8.1.0-1

yarn does not accept 4 digit 8.1.0.1
like it's used in tags for this project

bert-e commented:
# Hello bourgoismickael,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/werelogs/doc/user).





<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue WRLGS-11 contains:


* `7.10.5`

* `8.0.0`

* `8.1.5`



Considering where you are trying to merge, I expected to find at least:



* `8.1.0.1`


Please check the `Fix Version/s` of WRLGS-11, or the target
branch of this pull request.

BourgoisMickael commented:
ping

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

BourgoisMickael commented:
/approve

BourgoisMickael committed:
WRLGS-11: Fix coverage test in latest branch

nyc coverage seems to slow down the process on first run
The slow seems to vary between branches (due to nyc version)

BourgoisMickael commented:
ping

BourgoisMickael commented:
/approve

BourgoisMickael committed:
WRLGS-11: Copy GHA test workflow from dev/8.1

This is based on an old tag that had no GHA

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `hotfix/8.1.0`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.10`
* `development/7.4`
* `development/8.0`
* `development/8.1`



Please check the status of the associated issue WRLGS-11.

Goodbye bourgoismickael.




*The following options are set:* **approve**
